system/Validation/Validation.php

```diff
@@ -377,20 +377,29 @@ public function withRequest(RequestInterface $request): ValidationInterface
 	 *
 	 * @param string      $field
 	 * @param string|null $label
-	 * @param string      $rules
+	 * @param string|array $rules
 	 * @param array       $errors
 	 *
 	 * @return $this
 	 */
-	public function setRule(string $field, string $label = null, string $rules, array $errors = [])
+	public function setRule(string $field, string $label = null, $rules, array $errors = [])
```

Comment:
This is technically a BC but I'm fine with this as long we document this method signature change in the changelog.

Reply:
Is this a to do for me? If yes, where to document this? :)

Reply:
Kindly include an entry in user guide `changelogs/4.2.0.rst`

Reply:
Sadly there is no such file, I'm expecting it at `user_guide_src/source/changelogs/v4.2.0.rst`, am i at the wrong place?
Would add one if you're fine with that?

Reply:
I thought there's one made already. Kindly add one.